includes/collection/class-followers.php

```diff
@@ -292,14 +292,30 @@ public static function send_follow_response( $actor, $object, $user_id, $followe
 	/**
 	 * Get the Followers of a given user
 	 *
-	 * @param int    $user_id The ID of the WordPress User
-	 * @param string $output  The output format, supported ARRAY_N, OBJECT and ACTIVITYPUB_OBJECT
-	 * @param int    $number  Limts the result
-	 * @param int    $offset  Offset
-	 *
-	 * @return array The Term list of Followers, the format depends on $output
+	 * @param int    $user_id The ID of the WordPress User.
+	 * @param int    $number  Maximum number of results to return.
+	 * @param int    $page    Page number.
+	 * @param array  $args    The WP_Query arguments.
+	 * @return array List of `Follower` objects.
 	 */
 	public static function get_followers( $user_id, $number = -1, $page = null, $args = array() ) {
+		$raw = self::get_followers_raw( $user_id, $number, $page, $args );
+		return $raw['followers'];
+	}
+
+	/**
+	 * Get the Followers of a given user, along with a total count for pagination purposes.
+	 *
+	 * @param int    $user_id The ID of the WordPress User.
+	 * @param int    $number  Maximum number of results to return.
+	 * @param int    $page    Page number.
+	 * @param array  $args    The WP_Query arguments.
+	 *
+	 * @return array
+	 *               followers List of `Follower` objects.
+	 *               total     Total number of followers.
+	 */
+	public static function get_followers_raw( $user_id, $number = -1, $page = null, $args = array() ) {
```

Comment:
I think `raw` is a bit misleading, I would use a different function name something like `_with_count` or similar.